plugins/modules/firewalld.py

```diff
@@ -845,7 +847,7 @@ def main():
         if changed is True:
             msgs.append("Changed icmp-block %s to %s" % (icmp_block, desired_state))
 
-    if icmp_block_inversion is not None:
+    if icmp_block_inversion not in NEGATIVES:
```

Comment:
In this conditional branch, if `icmp-block-inversion` is `no`, the icmp-block-inversion setting seems to be skipped.
Shouldn't this process be skipped only if icmp_block_inversion is undefined?